modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/multiselect/MultiSelectEpisodesHelper.kt

```diff
@@ -190,7 +190,7 @@ class MultiSelectEpisodesHelper @Inject constructor(
 
             episodeManager.markAsUnplayed(list)
             episodeAnalytics.trackBulkEvent(AnalyticsEvent.EPISODE_BULK_MARKED_AS_UNPLAYED, source, list.size)
-            launch(Dispatchers.Main) {
+            withContext(Dispatchers.Main) {
```

Comment:
This change isn't really important at this point since we just launched a coroutine a few lines earlier, but the fewer times we launch new coroutines the fewer opportunities there are for race conditions, so I included this since it is a small step in the right direction.

Reply:
Great change. I just read about `withContext` and it makes sense not to kick off another Coroutine.